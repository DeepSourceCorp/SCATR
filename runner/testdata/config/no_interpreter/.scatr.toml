test_checks = false
test_autofix = false

[checks]
script = "script"

[autofix]
script = "script"
