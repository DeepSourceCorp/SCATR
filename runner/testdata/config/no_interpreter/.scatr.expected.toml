test_checks = false
test_autofix = false

[checks]
script = "script"
interpreter = "sh"

[autofix]
script = "script"
interpreter = "sh"

[processor]
interpreter = "sh"
